src/SprykerEco/Zed/UnzerApi/Business/Api/Request/Converter/AuthorizableChargeRequestConverter.php

```diff
@@ -22,7 +22,7 @@ public function convertRequestTransferToArray(UnzerApiRequestTransfer $unzerApiR
         $unzerApiChargeRequestTransfer = $unzerApiRequestTransfer->getChargeRequestOrFail();
 
         return [
-            UnzerApiRequestConstants::PARAM_AMOUNT => $unzerApiChargeRequestTransfer->getAmountOrFail(),
+            UnzerApiRequestConstants::PARAM_AMOUNT => (string)$unzerApiChargeRequestTransfer->getAmountOrFail(),
```

Comment:
You should mention all transfer mandatory properties in Public API specification `UnzerApiFacadeInterface`.
Please, check all converters usage.

Reply:
done